cmd/minikube/cmd/start_flags.go

```diff
@@ -457,6 +457,18 @@ func getCNIConfig(cmd *cobra.Command) string {
 	return chosenCNI
 }
 
+func getNetwork(driverName string) string {
+	n := viper.GetString(network)
+	if n != "" {
+		return n
+	}
+	if driver.IsQEMU(driverName) {
+		out.WarningT("The default network for QEMU will change from 'user' to 'socket' in a future release")
```

Comment:
lets add a website entry for scoket_vmnet
and talk about how it is different and how to install it

also lets add this to the message

"You are using qemu driver without a dedicated network, which does not have support for some networking commands such as 'minikbue service
 or 'minikube tunnel' if you like to try the experimental dedicated driver checkout the instructions on ...URL..."

Reply:
Added a website entry describing the two different network options and how to install socket_vmnet.

Also added the CLI message to warn users when they're starting QEMU without a dedicated network.